### ISSUE TITLE
Crash when opening world created with unstable with stable release

### ISSUE THREAD

**Issue by Forecaster** [ForestryMC/ForestryMC#724]
I have been changing forestry versions around after finding that the version I was using wasn't updated for the forge fluid changes. I encountered some issues where the server wouldn't load the world. I had to load the world without forestry, which then crashed due to bccompat. I disabled that too and ran it without either. It loaded up and I shut it down and re-enabled forestry and bccompat and everything seemed to work. I then booted up my client with the same configuration and created a new world with the intention of testing something, but the world then crashed with this crash: http://hastebin.com/ezivolisox.pl
Which appears to be an incompatability with neiintergration, probably because something changed in unstable.
I then installed the latest stable (3.5.7.16) and tried opening the world I made which resulted in this crash: http://hastebin.com/ruxolulogo.vhdl
Which is the focus of this report.

TLDR; Reverting from 3.6.0.588-unstable to 3.5.7.16 (stable) resulted in this crash: http://hastebin.com/ruxolulogo.vhdl
when trying to open a world created with the unstable version.

**Comment by mezz** [comment]
Thanks for the report!
NEI Integration has has a beta version for Forestry unstable builds. 
You can grab it in the ["Other Downloads" section here](http://www.curse.com/mc-mods/minecraft/225251-nei-integration#t1:other-downloads).

You will not be able to downgrade properly from unstable builds, it will crash. Please always back up your world when you update with unstable. That said, you will be just fine with an updated NEI Integration! Please report back if you have any more issues.
